src/js/app.jsx

```diff
@@ -1,22 +1,25 @@
 import React from 'react';
-import Perf from 'react-addons-perf';
```

Comment:
This change should have been in #1074.

See react-documentation about this pacakge being no-longer supported effective w/ react 16: https://reactjs.org/docs/perf.html